controls/listpicker/commands.md

```diff
@@ -9,29 +9,31 @@ slug: listpicker-commands
 
 # .NET MAUI ListPicker Commands
 
+This article described the commands available in the Telerik UI for .NET MAUI ListPicker control. 
+
 ## ListPicker Commands
 
 List Picker for .NET MAUI exposes the following commands you can use to programmatically manipulate displaying the popup as well as clearing the selected item:
 
 * `ToggleCommand`(`ICommand`)&mdash;Allows you to show/hide the popup used for selecting an item from a list of items.
 * `ClearCommand`(`ICommand`)&mdash;Allows you to clear the displayed item.
 
-## PopupSelector Commands
+## Settings Commands
```

Comment:
Please see the DatePicker or the DateTimePicker for the suggested changes in this section.